x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/SearchableSnapshotsRollingUpgradeIT.java

```diff
@@ -227,6 +227,19 @@ private void executeBlobCacheCreationTestCase(Storage storage, long numberOfDocs
             assertHitCount(index, equalTo(numberOfDocs * 2L));
             deleteIndex(index);
 
+            if (UPGRADE_FROM_VERSION.onOrAfter(Version.V_8_0_0)) { // TODO Adjust to 7.13.0
+                final Request request = new Request("GET",
+                    "/.snapshot-blob-cache/_settings/index.routing.allocation.include._tier_preference");
+                request.setOptions(expectWarnings("this request accesses system indices: [.snapshot-blob-cache], but in a future major " +
+                    "version, direct access to system indices will be prevented by default"));
+
+                final Map<String, ?> snapshotBlobCacheSettings = entityAsMap(adminClient().performRequest(request));
+                assertThat(snapshotBlobCacheSettings, notNullValue());
+
+                final String tierPreference = (String) snapshotBlobCacheSettings.get("");
+                assertThat(tierPreference, nullValue());
```

Comment:
I am not sure I understand this. I would have thought it should validate that the setting was `data_content,data_hot`?

Reply:
You don't understand because it's plain wrong, a change did not made it to the commit. I pushed ea2b40a4b4f5e7fca012a6b973dd298008c1d7f7